hawk/Gemfile

```diff
@@ -26,15 +26,7 @@ gem "gettext_i18n_rails", ">= 1.8"
 gem "gettext_i18n_rails_js", ">= 1.3"
 gem "sprockets", ">= 3.7"
 gem "kramdown", ">= 1.14"
-
-group :development do
```

Comment:
@MalloZup I don't think it is really needed, as this are gem used only if you run bundler with the development mode... so these are useful for debug or development.. but does not affect HAWK build on the default or production setup.

Reply:
this is needed because OBS try to build such gems from dev. And we have a dirty patch for this.